### PR TITLE
Bra 14 homework 02

### PR DESCRIPTION
**Pull request by dev-kucha**
Lesson 14
Part 2

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+    letter-spacing: 0.5rem;
+    max-width: 590px;
+    margin: auto;
+    margin-bottom: 5.3rem;
+}
+
+.header__site-subtitle {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.5rem;
+    font-weight: 400;
```
Comment (roskyba) [inline css/styles.css]: 400 is the default value for `font-weight`. Same as `normal`

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+.footer__site-icon {
+    width: 3.5rem;
+    height: 3.7rem;
+    fill: #3C3C3C;
+}
+
+.footer__copyrite {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.4rem;
+    font-weight: 400;
```
Comment (roskyba) [inline css/styles.css]: 400 is the default value for `font-weight`. Same as `normal`

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+    text-transform: uppercase;
+    color: #ffffff;
+    left: 3rem;
+    bottom: 2rem;
+}
+
+.btn-submit {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.6rem;
+    font-weight: 400;
```
Comment (roskyba) [inline css/styles.css]: 400 is the default value for `font-weight`. Same as `normal`

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+            -ms-flex-negative: 1;
+        flex-shrink: 1;
+    }
+
+    .btn-submit {
+        padding: 2.4rem 8.7rem;
+    }
+}
+
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline css/styles.css]: Devices less than 320px no need to be supported in responsive. This rule can be removed. Or change to `min-width: 320px`

**Comment by dev-kucha** [comment]
All found bugs are fixed 22-03-2021 20:33

**Review by roskyba — APPROVED** [review]

**Comment by dev-kucha** [comment]
Done

**Pull request closed without merging**